Remotion/Development/Moq/UnitTesting/Serializer.Partial.cs

```diff
@@ -18,7 +18,7 @@
 
 namespace Remotion.Development.UnitTesting
 {
-  public partial class Serializer
+  public static partial class Serializer
```

Comment:
Drop. Development.Moq already references Development.Core, that way, it gets access to the Serializer type needed by SerializationCallbackTester.